includes/adapter/OrderLineFromWooCommerce.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\WooCommerce\Includes\Adapter;
+
+use MyParcelNL\Sdk\src\Model\Fulfilment\OrderLine;
+use WC_Order_Item;
+
+class OrderLineFromWooCommerce extends OrderLine
+{
+    public function __construct(WC_Order_Item $wcOrderItem)
+    {
+        $standardizedDataArray = $this->prepareItemData($wcOrderItem);
+
+        parent::__construct($standardizedDataArray);
+
+    }
+
+    /**
+     * @param \WC_Order_Item $wcOrderItem
+     *
+     * @return array
+     */
+    protected function prepareItemData(WC_Order_Item $wcOrderItem): array
+    {
+        $wcItemData = $wcOrderItem->get_data();
+
+        $data                    = [];
+        $price                   = (int) ($wcItemData['subtotal'] * 100.0);
+        $vat                     = (int) ($wcItemData['subtotal_tax'] * 100.0);
+        $data['price']           = $price;
+        $data['vat']             = $vat;
+        $data['price_after_vat'] = $price + $vat;
+        $data['quantity']        = $wcOrderItem->get_quantity();
+        $data['product']         = $this->prepareProductData($wcOrderItem);
+        // $data['instructions']
+        // $data['uuid']
+
+        return $data;
+    }
+
+    /**
+     * @param \WC_Order_Item $wcOrderItem
+     *
+     * @return array
+     */
+    protected function prepareProductData(WC_Order_Item $wcOrderItem): array
+    {
+        $wcItemData = $wcOrderItem->get_data();
+        $wcProduct  = $wcOrderItem->get_product();
+
+        $data                        = [];
+        $data['external_identifier'] = (string) ($wcItemData['variation_id'] ?: $wcItemData['product_id']);
+        $data['name']                = $wcItemData['name'];
+        $data['sku']                 = $wcProduct->get_sku();
+        $data['height']              = (int) $wcProduct->get_height() ?: 0;
+        $data['length']              = (int) $wcProduct->get_length() ?: 0;
+        $data['weight']              = (int) $wcProduct->get_weight() ?: 0;
+        $data['width']               = (int) $wcProduct->get_width() ?: 0;
+        $data['description']         = $wcProduct->get_short_description();
+        // $data['ean'] // GTIN only via plugins, do we want to handle that?
+        // $data['uuid']
+
+        return $data;
+    }
+}
```

Comment:
Je mist hier een newline, dat is wel gek, normaal fixt phpstorm dat automatisch in alle bestanden die je opent